<launch>  
  <include file="$(find agile_grasp_filter)/launch/single_camera_grasps.launch">
    <!--<param name="frame" value="/camera_rgb_optical_frame"/>-->
    <arg name="num_samples" value="1000"/>
    <arg name="hand_outer_diameter" value="0.15"/>
  </include>

  <node name="agile_grasp_filter" pkg="agile_grasp_filter" type="agile_grasp_filter_node" output="screen"> 
    <param name="threshold" value="0.8"/> <!-- [-1.0 , 1.0] -->
  </node> 
</launch>